src/PositionManager/PositionManager.h

```diff
@@ -21,7 +21,7 @@ class QGCPositionManager : public QGCTool {
     Q_OBJECT
 
 public:
-
+    static constexpr size_t MinHorizonalAccuracyMetres = 100;
```

Comment:
MinHorizonalAccuracy**Meters**, just a missing typo.

Reply:
it's just British - force of habit. 
I count metres:
```
62 matches across 16 files
```
vs meter
```
1276 matches across 98 files
```
- ok - i am in the wrong it seems.

Reply:
Done https://github.com/mavlink/qgroundcontrol/pull/9483/commits/21d180fb100ffbc92ed842f4551029b3e52728b1